backend/Car-Detailing-Backend/Data/ReadingAndWriting.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Car_Detailing_Backend.Data.Interface;
+
+namespace Car_Detailing_Backend.Data
+{
+    public class ReadingAndWriting : IReadingAndWriting
+    {
+        public async Task<List<string>> ReadingThingsOnFileAsync(string filePath)
+        {
+            try
+            {
+                var data = new List<string>();
+                var file = await File.ReadAllLinesAsync(filePath);
+                foreach (var line in file)
+                {
+                    data.Add(line);
+                }
+                return data;
+            }
+            catch (Exception)
+            {
+                throw;
+            }
```

Comment:
`Try ... catch` is redundant here, because you still throw an exception and don't handle it here. Please Check other similar cases an remove unused `Try ... catch`

Reply:
The code catch (Exception) { throw; } is used to catch an exception of any Exception type and to report this exception again. This means that the exception is handled temporarily in the catch block, but is then reported again to continue handling the exception at a higher level of the call hierarchy.

If you think we don't need to have this kind of protection let me know and I will modify the code

Reply:
I think it's not protection because you still throw the caught exception further from the lower part. I would recommend only having `Try .. catch` in controllers because there you are handling an error. 

My IDE also suggests, that this type of `Try ... catch` connected to this comment is redundant
![image](https://github.com/pploszczyca/Car-Detailing/assets/37903018/6228e35a-a35b-47fc-8b2e-bddc1cca34a6)

And it will be the same as
![image](https://github.com/pploszczyca/Car-Detailing/assets/37903018/8a5ca348-02be-4d7d-9ec1-337ee5fa40cb)